### ISSUE TITLE
Integration tests fail because of wrong module sequence

### ISSUE THREAD

**Issue by domeglic** [PAYONE-GmbH/magento-2#95]
When running integration tests on a project with this module it will fail with the message:

> Base table or view not found: 1146 Table 'integration_test.quote_address' doesn't exist, query was: DESCRIBE `quote_address`

Since there is no sequence defined, this modules setup is run very early and fails because quote tables have not been created yet. 
Adding **Magento_Quote** to the sequence fixes the problem (Should probably also add **Magento_Sales** since order tables are being changed to be more correct).

Would be really helpful since it should not break anything and there is no way to add a sequence to another module as far as I know.

**Comment by fjbender** [comment]
Thanks for the feedback!

Just to be sure: You're referencing to the `<sequence>` Tags in our `module.xml` file as defined in http://devdocs.magento.com/guides/v2.0/extension-dev-guide/build/module-load-order.html, right?

Altering the file this way would suffice?

```xml
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../lib/internal/Magento/Framework/Module/etc/module.xsd">
    <module name="Payone_Core" setup_version="1.3.4">
        <sequence>
            <module name="Magento_Quote" />
            <module name="Magento_Sales" />
        </sequence>
    </module>
</config>
```

**Comment by fjbender** [comment]
@domeglic Can you confirm that the integration tests run with my changes from #96?

**Comment by domeglic** [comment]
I have tested it and can confirm that fix #96 solves the problem.
